draft-ietf-rats-corim.md

```diff
@@ -1345,7 +1345,7 @@ If the search criteria are satisfied, the `endorsements` entries are asserted wi
 #### Conditional Endorsement Series Triple {#sec-comid-triple-cond-series}
 
 The Conditional Endorsement Series Triple is used to assert endorsed values based on an initial condition match (specified in `condition:`) followed by a series condition match (specified in `selection:` inside `conditional-series-record`).
-Every `conditional-series-record` selection MUST select the same mkeys where every selected mkey's corresponding set of code points (i.e., mval.key) MUST be the same across each `conditional-series-record`.
+Every `conditional-series-record` selection MUST select the same mkeys where every selected mkey's corresponding set of code points MUST be the same across each `conditional-series-record`.
```

Comment:
```suggestion
Every `conditional-series-record` selection MUST select the same mkeys where every selected mkey's corresponding set of code points ( i.e., mval.key )MUST be the same across each `conditional-series-record`.
```

Reply:
What? No, the point is to remove parentheses from normative statements.

Reply:
The item in the paranthesis is a highlight to remove the confusion as the code point is for mval.key rather than a mkey in a measurement-map, so it is super important to have it here. I am ok, to remove the paranthesis around it, if it becomes a necessity!